Tests/StencilSwiftKitTests/ParametersTests.swift

```diff
@@ -93,6 +93,17 @@ class ParametersTests: XCTestCase {
                    "The order of arrays are properly preserved when flattened")
   }
 
+  func testFlattenBool() {
+    let trueFlag = Parameters.flatten(dictionary: ["test": true])
+    XCTAssertEqual(trueFlag, ["test"], "True flag is flattened to a param without value")
+
+    let falseFlag = Parameters.flatten(dictionary: ["test": false])
+    XCTAssertEqual(falseFlag, [], "False flag is flattened to nothing")
+
+    let otherFlag = Parameters.flatten(dictionary: ["test": "a"])
+    XCTAssertEqual(otherFlag, ["test=a"], "Non-boolean flag is flattened to a parameter with value")
+  }
```

Comment:
Should we add tests that `Parameters.flatten(dictionary: ["test": "false"]) still passes the string `"false"`) along?